src/components/sendNew/send.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import Box from '../box';
+import MultiStep from './../multiStep';
+import ResultBox from '../resultBox';
+import Form from './steps/form';
+import Confirm from './steps/confirm';
+import FollowAccount from '../sendTo/followAccount';
+import PassphraseSteps from './../passphraseSteps';
+import AccountInitialization from '../accountInitialization';
+import { parseSearchParams } from './../../utils/searchParams';
+import breakpoints from './../../constants/breakpoints';
+import styles from './send.css';
+
+class Send extends React.Component {
+  constructor(props) {
+    super(props);
+    const needsAccountInit = !props.account.serverPublicKey
+      && props.account.balance > 0
+      && props.pendingTransactions.length === 0;
+
+    const { amount, recipient } = this.getSearchParams();
+    this.state = {
+      isActiveOnMobile: !!recipient || !!amount || needsAccountInit,
+      isActiveTabSend: true,
+    };
+  }
+
+  getSearchParams() {
+    return parseSearchParams(this.props.history.location.search);
+  }
+
+  setActiveOnMobile({ isActiveOnMobile, isActiveTabSend = true }) {
+    this.setState({ isActiveOnMobile, isActiveTabSend });
+  }
+
+  setActiveTabSend(isActiveTabSend) {
+    this.setState({ isActiveTabSend });
+  }
```

Comment:
`setActiveTabSend` is most likely also not needed anymore

Reply:
I forget to remove this before but I just removed it